com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -60,88 +60,41 @@ define([
-              'option_group_id': '111',
-              'label': 'Senior Manager',
-              'value': 'Senior Manager',
-              'name': 'Senior_Manager',
-              'is_default': '0',
-              'weight': '1',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '846',
-              'option_group_id': '111',
-              'label': 'Junior Manager',
-              'value': 'Junior Manager',
-              'name': 'Junior_Manager',
-              'is_default': '0',
-              'weight': '2',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '879',
-              'option_group_id': '124',
-              'label': 'Other',
-              'value': 'Other',
-              'name': 'Other',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '3',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '1045',
-              'option_group_id': '124',
-              'label': 'Test A',
-              'value': '1',
-              'name': 'Test A',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '2',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '1046',
-              'option_group_id': '124',
-              'label': 'Test B',
-              'value': '2',
-              'name': 'Test B',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '1',
-              'description': 'Test B',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            }
-          ],
-          'optionGroupData': {
-            'cost_centres': '124'
-          }
-        }));
+        spyOn(jobRoleService, 'getOptionValues').and.returnValue($q.resolve(Mock.option_values));
         initController();
         $rootScope.$digest();
       });
 
-      it('builds the "CostCentreList" array of objects containing the "weight" property', function () {
-        expect(ctrl.CostCentreList.length).toBe(3);
+      it('builds the "CostCentreList" collection containing the "weight" property', function () {
+        expect(Object.keys(ctrl.CostCentreList).length).toBe(3);
         expect(ctrl.CostCentreList[0].weight).not.toBeNull();
         expect(ctrl.CostCentreList[1].weight).not.toBeNull();
         expect(ctrl.CostCentreList[2].weight).not.toBeNull();
       });
+
+      it('builds the "hrjc_level_type" collection containing the "value" property', function () {
+        expect(Object.keys(ctrl.LevelsData).length).toBe(1);
+        var levelValue = Mock.option_values.values[0].value;
```

Comment:
The variable name `Mock` is not really descriptive, may we change it to `JobRoleDataMock` or smth like that?